elyra/pipeline/kfp/kfp_authentication.py

```diff
@@ -628,7 +633,7 @@ class K8sServiceAccountTokenAuthenticator(AbstractAuthenticator):
 
     def authenticate(self,
                      kf_endpoint: str,
-                     runtime_config_name: str) -> None:
+                     runtime_config_name: str) -> TokenCredentialsBase:
```

Comment:
Can you please also update the doc string (`:return: ...`) to reflect the change?

Reply:
Fixed in https://github.com/elyra-ai/elyra/pull/2358/commits/0c354d5ba8fe7c31026f9a11f6a0a3d0a6ab42a0